Solutions/moritz_w/Lesson_9/set_and_tuple.py

```diff
@@ -0,0 +1,26 @@
+import unittest
+import string
+
+
+def not_present_letters(text):
```

Comment:
Consider this :) :
def not_present_letters(text):
    return set(string.ascii_letters).difference(set(text))